src/hotspot/share/gc/shenandoah/shenandoahScanRemembered.cpp

```diff
@@ -939,10 +933,7 @@ void ShenandoahReconstructRememberedSetTask::work(uint worker_id) {
         oop obj = cast_to_oop(obj_addr);
         size_t size = obj->size();
 
-        // First, clear the remembered set for all spanned humongous regions
```

Comment:
In fact, we should check if card scanning actively clears the read table. If it doesn't we need to make sure it does or otherwise make sure that card marks do not accumulate monotonically in the table.